tests/libcxx/enc/CMakeLists.txt

```diff
@@ -35,6 +35,12 @@ target_include_directories(libcxxtest-support PRIVATE
         ${CMAKE_CURRENT_BINARY_DIR}
         )
 
+# Log the build information.
+set(BUILD_INFO_FILE "${CMAKE_CURRENT_BINARY_DIR}/build_info.cmake")
+file(WRITE "${BUILD_INFO_FILE}" "")
+file(APPEND "${BUILD_INFO_FILE}" "set(LINUX_CXX_COMPILER_ID \"${CMAKE_CXX_COMPILER_ID}\")\n")
```

Comment:
Nonblocking: You could probably save yourself a line and `WRITE` this first line of content into the file instead of writing an empty string then appending.